data/projects/extend_linter_integration.md

```diff
@@ -35,6 +33,143 @@ to include the newly written Linter bears.
+attributes, report their difference or any missing attribute, specifying the
+equality mismatch.
+In the case of `execute_bear(...)`, I would like to print `stdout` and `stderr`
+from linter bears when tests fail. Often, our test suite breaks due to upstream
+releases and we have to go through the hassle of detecting the breakage and
+brief results without sufficient explanation requires a ton of work to be done.
+The given implementation is expected to eliminate the usage of a debugger,
+providing productive results while writing and testing bears.
+
+To summarise, the new Testing API would look like:
+![new test api](https://cloud.githubusercontent.com/assets/14862976/26557881/4103628c-44c2-11e7-9a06-68591d000aa0.jpg)
+
+**(B) PROBLEM: PLENTY OF ISSUES RELATED TO THE MARKDOWNBEAR**
+
+Markdown is a very crucial format as nearly every repository has some files
+with a .md extension and many users may go disappointed since the Markdown
+bear has a considerable number of issues related to it, the most important
+being that the patch proposes to delete the entire file. Some more issues
+related to it are:
+
+**ISSUE:** [MarkdownBear doesn’t check for trailing spaces](https://github.com/coala/coala-bears/issues/1179)
+
+**APPROACH:** I plan to eliminate this issue by integrating this plugin
+(https://www.npmjs.com/package/remark-lint-no-trailing-spaces) which is
+expected to remove trailing whitespaces for all cases. The plugin will be
+installed via package.json and then respectively passed on as shell argument
+to the remark tool.
+
+**ISSUE:** [Markdown parsing issues](https://github.com/coala/coala-bears/issues/940)
+
+**APPROACH:** Likely to be solved by Use `remark-lint` to report meaningful
+messages since all errors will be reported by that commit.
+
+**ISSUE:** [Use remark-lint to report meaningful messages](https://github.com/coala/coala-bears/issues/926) (critical)
+
+**APPROACH:** This patch will be vital and aims to use the `remark-lint` plugin
+to report fruitful errors instead of tidying the markdown automatically, because
+in CI mode, the tidying of the file is useless and descriptive messages are
+critical. Since it is already being included by package.json, implementing
+it won’t be a difficult task.
+
+**ISSUE:** [Add remark-validate-links plugin](https://github.com/coala/coala-bears/issues/924)
+
+**APPROACH:** This commit will add a plugin that checks that the local files exist
+relative to the markdown
+file(https://www.npmjs.com/package/remark-validate-links). The plugin will be
+installed via package.json and then respectively passed on as shell argument
+to the remark tool or in the config file, depending on the requirement.
+
+**ISSUE:** [MarkdownBear table settings](https://github.com/coala/coala-bears/issues/1008)
+
+**APPROACH:** Likely to be solved by Use `remark-lint` to report meaningful
+messages since all errors will be reported by the patch.
```

Comment:
Nope, See https://github.com/coala/coala-bears/issues/1008#issuecomment-261224591 and https://github.com/coala/coala-bears/issues/940#issuecomment-256615428. The [same patch](https://github.com/coala/coala-bears/issues/926) is likely to solve both the issues. :wink: